pkg/cache/cache_test.go

```diff
@@ -715,9 +715,11 @@ func TestCacheWorkloadOperations(t *testing.T) {
 		*utiltesting.MakePodSet("driver", 1).
 			Request(corev1.ResourceCPU, "10m").
 			Request(corev1.ResourceMemory, "512Ki").
+			TotalRequestFunc(workload.PodTotalRequests).
```

Comment:
Is there an scenario where we pass a different function?

If this is about avoiding a circular dependency, it might be better to call `workload.PodSetsTotalRequests` once in the test body function, instead of doing it in the wrappers.

Reply:
no, but workload package is also using the wrappers and I'm doing this to avoid circular dependencies

Reply:
Right. Then there are two alternatives:
1. Avoid making use of the wrappers in `workload_test.go`. These tests are small and intend to test multiple PodSpec fields that might not be worth to add in the wrappers.
2. Call some form of `workload.PodSetsTotalRequest` from the test body instead of in the wrappers.

Reply:
no longer needed